src/pivotal-ui-react/helpers/helpers.js

```diff
@@ -0,0 +1,18 @@
+import classnames from 'classnames';
+
```

Comment:
Since we are publishing this module as a "react component" (i.e. it is `pui-react-helpers`, we need to have javadocs for this component. See https://github.com/pivotal-cf/pivotal-ui/blob/master/CONTRIBUTING.md#readme-documentation---react-components